backend/src/main/java/gov/cdc/usds/simplereport/service/DeviceTypeSyncService.java

```diff
@@ -135,7 +135,7 @@ public class DeviceTypeSyncService {
 
   public String extractSpecimenTypeCode(String specimenDescription) {
     Pattern specimenCode = Pattern.compile("^(.*?)\\^");
-    return extractSpecimenDetails(specimenDescription, specimenCode);
+    return extractSpecimenDetails(specimenDescription, specimenCode).strip();
```

Comment:
We needed to run a device sync to put the equipment UID type into our dev 2 db, but turns out there was an extra white space char in one of the specimen types coming from the LIVD API that was blowing up the sync (since that field expected only number values)

Adding this here to protect against it, but lmk if we want to make this more robust / if we should follow up with RS

<img width="1383" alt="Screenshot 2023-10-27 at 3 45 13 PM" src="https://github.com/CDCgov/prime-simplereport/assets/29645040/da83ec5c-4a19-482d-8454-ba90b8e86dd0">